kpi/models/import_export_task.py

```diff
@@ -462,10 +468,12 @@ def _build_export_options(self, pack):
         group_sep = self.data.get('group_sep', '/')
         translations = pack.available_translations
         lang = self.data.get('lang', None) or next(iter(translations), None)
+        lang_header = self.data.get('lang_header', lang) or next(iter(translations), None)
```

Comment:
```suggestion
        lang_header = self.data.get('lang_header', lang)
```

because `lang` is the fallback value, and it's set in the line above, we don't need a second fallback value